release/pokerth.xml

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2015 Andrew Ziem
+    http://bleachbit.sourceforge.net
```

Comment:
Done for all cleaners https://github.com/az0/cleanerml/commit/55a24dbc7fad1a001fa1b0b417ad53da093fae5c